docs/root/start/sandboxes/gzip.rst

```diff
@@ -0,0 +1,128 @@
+.. _install_sandboxes_gzip:
+
+Gzip
+====
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+        Used to make ``HTTP`` requests.
+
+Enable compression in Envoy would save some bandwidth.
+
+Step 1: Start all of our containers
+***********************************
+
+Change to the ``examples/gzip`` directory.
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/gzip
+    $ docker-compose build --pull
+    $ docker-compose up -d
+    $ docker-compose ps
+              Name                  Command               State                                               Ports
+    ----------------------------------------------------------------------------------------------------------------------------------------------------
+    gzip_gzip_1   /docker-entrypoint.sh /usr ... Up      10000/tcp, 0.0.0.0:8001->8001/tcp,:::8001->8001/tcp, 0.0.0.0:8002->8002/tcp,:::8002->8002/tcp
+
+Step 2: Test Envoy's stats capabilities
+***************************************
```

Comment:
can you move this to after the examples working with files - same in the config and the verify.sh script

Reply:
fine.